lib/plugins/inspec-suggest/lib/inspec-suggest/suggtool/base.rb

```diff
@@ -55,6 +55,14 @@ def with_matching_files(set_cfg, &block)
         end
       end
 
+      def with_matching_yaml_files(set_cfg, &block)
+        set_cfg["paths"].each do |path|
+          Dir.glob(File.join(@working_dir, path, "inspec.yml")).each do |control_file_path|
+            yield(control_file_path)
```

Comment:
This is not iterating on control_files. Please rename the iterator variable to be something like "metadata_file"